sdk/storage/azblob/appendblob/client.go

```diff
@@ -34,7 +34,7 @@ type Client base.CompositeClient[generated.BlobClient, generated.AppendBlobClien
 //   - cred - an Azure AD credential, typically obtained via the azidentity module
 //   - options - client options; pass nil to accept the default values
 func NewClient(blobURL string, cred azcore.TokenCredential, options *ClientOptions) (*Client, error) {
-	authPolicy := runtime.NewBearerTokenPolicy(cred, []string{shared.TokenScope}, nil)
```

Comment:
does it make sense to make this TokenScope also a configurable option. Not sure in what condition user will need it but having it as an option in the API means there are cases where user want to override this.

Reply:
@jhendrixMSFT is that 

> Is it possible to add a live/recorded test at least for page blobs to test this?

A live/recorded test might be hard since the Disk needs a specific role (not sure how to do this programmatically) and then we have to invoke the compute management plane SDK. I've added a comment of my test and what is required for the test to work. The mock tests should be sufficient

Reply:
I think this might go against the original intent of the Identity SDK. It seems like exposing TokenScope is a decision the Identity SDK should consider, not us.

Reply:
Fact that we are able to pass this as input while creating new policy, I guess identity already has exposed it. We are not giving user a choice to pass this as an argument and hardcoding it here.

Reply:
Identity/core has to expose it to SDK devs, which they did. 

Additionally, none of the other platforms support setting the token scope, so I don't think we need to be reinventing the wheel here.